src/full/Agda/Interaction/JSONTop.hs

```diff
@@ -14,6 +14,7 @@ import Agda.Interaction.Base
   (CommandState(..), CurrentFile(..), ComputeMode(..), Rewrite(..), OutputForm(..), OutputConstraint(..))
 import qualified Agda.Interaction.BasicOps as B
 import Agda.Interaction.EmacsTop
+import Agda.Interaction.Imports (getAllWarningsOfTCErr)
```

Comment:
I suspect my recently recently-merged #5067 may cause a build break here: the import will need to be `import Agda.TypeChecking.Errors (prettyError, getAllWarningsOfTCErr)` (to save you time tracking that down!)